ledger/participant-integration-api/src/test/lib/scala/platform/indexer/ha/IndexerStabilitySpec.scala

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
how about adding a test case which proves that if no Ha protection, the test fails?
(for example with H2 we might be able to implement such, because that has no mutual exclusion on the indexer)
* it would be super nice to see that the integrity check fails
* also it would be nice to see that we have several indexers indexing

Reply:
Added a H2 test. I have refactored the suite so that it works with H2 as well, and changed the integrity checking part so that the H2 test asserts that the integrity check fails with duplicate event sequential ids.